terraform/staging.tfvars

```
@@ -7,6 +7,28 @@ ssm_deployment_parameters_path_prefix = "/gost/staging/deploy-config"
 // Website
 website_enabled     = true
 website_domain_name = "staging.grants.usdr.dev"
+website_managed_waf_rules = {
+  "AnonymousIpList" = {
+    managed_rule      = "AWSManagedRulesAnonymousIpList",
+    priority          = 5,
+    metric_visibility = false
+  },
+  "AmazonIpReputationList" = {
+    managed_rule      = "AWSManagedRulesAmazonIpReputationList",
+    priority          = 10,
+    metric_visibility = false
+  },
+  "CommonRuleSet" = {
+    managed_rule      = "AWSManagedRulesCommonRuleSet",
+    priority          = 20,
+    metric_visibility = true
+  },
+  "KnownBadInputsRuleSet" = {
+    managed_rule      = "AWSManagedRulesKnownBadInputsRuleSet",
+    priority          = 30,
+    metric_visibility = true
+  }
+}
```

Do you think there's a benefit to having separate `managed_rule` attribute here? Would something like this work?
```suggestion
website_managed_waf_rules = {
  "AWSManagedRulesAnonymousIpList" = {
    priority          = 5,
    metric_visibility = false
  },
  "AWSManagedRulesAmazonIpReputationList" = {
    priority          = 10,
    metric_visibility = false
  },
  "AWSManagedRulesCommonRuleSet" = {
    priority          = 20,
    metric_visibility = true
  },
  "AWSManagedRulesKnownBadInputsRuleSet" = {
    priority          = 30,
    metric_visibility = true
  }
}
```

---

Yeah, I thought about that but AWS considers `name` separate from the actual name of the managed rule set you are selecting.  The `name` attribute is what is checked for uniqueness (thus you can have several different rules using the same managed `AWSManagedRulesAnonymousIpList` ruleset).  The use case here is that you can deploy several different versions the the ruleset all with different parameters (most of which I'm not exposing here since I'm not sure that lever of granularity is what we need at the moment).

---

Ah, gotcha- makes sense. Carry on!